src/tribol/mesh/CouplingScheme.cpp

```diff
@@ -962,6 +997,8 @@ bool CouplingScheme::init()
    valid = this->isValidCouplingScheme();
    if (valid)
    {
+      // set individual coupling scheme logging level
+      this->setSlicLoggingLevel();
```

Comment:
It wasn't clear to me why this would be called in an `init()` method until I saw where `init()` was being called.  Maybe renaming the method to something like `setupBeforeApply()` might be clearer?

Reply:
Leaving for now.